### DISCUSSION TITLE
Getting Error when building first app in Next.js

### DISCUSSION THREAD

**Discussion by jsobon12** [vercel/community#2549] [Help] [answered]
### Summary

Hey, I'm getting the below error when building my first app with Next.js. QQ's, 1. Is my file structure correct? (is my _App.js file in the right location) 2. Is the code in my _App.js file correct? Thanks.
![Screen Shot 2023-05-22 at 10 34 38 AM](https://github.com/vercel/community/assets/46903417/abc076e2-2b81-423d-adf5-d35b1865525e)

![Screen Shot 2023-05-22 at 10 28 16 AM](https://github.com/vercel/community/assets/46903417/88edad5b-68dc-4c3c-91a7-07552a037b7a)

![Screen Shot 2023-05-22 at 10 33 13 AM](https://github.com/vercel/community/assets/46903417/a6196185-88ef-424e-a52b-550735a00051)

**Comment by amyegan** [comment, marked as answer]
Hi @jsobon12! It looks like your `_app.js` is in `/pages/posts`, but it should be moved up to `/pages`.

**Comment by z4nr34l** [comment]
In `_app.js` change first line from:
```
import '../styles/global.css'
```
to:
```
import '../styles/globals.css'
```
or:
```
import '../global.css'
```

As you have 2 CSS files that are named similarly - so you made a typo, but IDK which one is your preffered to give exact solution :D

Hope it helped, please consider marking this solution as answer if it helped to help others find it.
